daffodil-cli/src/test/resources/clitests.tdml

```diff
@@ -1,35 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There's a single test that uses the TDMLRunner to execute this test. No test using the CLI actually uses this file, so this really doesn't belong in the daffodil-cli project and doesn't really add anything that isn't already tested, so just delete it.